tests/end_to_end/remoteTestDataVariants/variants/remoteTest/urls.json

```diff
@@ -0,0 +1,5 @@
+{
+    "urls": [
+        "http://0.0.0.0:8002/example_2.vcf.gz"
```

Comment:
1kgPhase1 and 1kgPhase3 would be a good place to start. If they don't work, then we are seriously in trouble.